include/swift/ABI/Metadata.h

```diff
@@ -3683,6 +3683,20 @@ struct TargetResilientSuperclass {
   TargetRelativeDirectPointer<Runtime, const void, /*nullable*/true> Superclass;
 };
 
+template <typename Runtime>
+struct TargetObjCResilientClassStub {
+  /// A relative pointer to an Objective-C resilient class stub:
+  /// - isa pointer, always 1
+  /// - an update callback, of type 'Class (*)(Class *, objc_class_stub *)'
```

Comment:
Is this a description of the pointee or something, and if so is there a reason we can't define this structure?  I assume we just don't currently need to interpret it within the Swift runtime.

At any rate, you should clarify that the pointee structure "belongs" to the ObjC ABI.

This structure's name suggests that it's the stub, but it's actually just a pointer to the stub, and the `getStub()` accessor looks through it.  The accessor behavior seems reasonable; you should probably call this `ObjCResilientClassStubInfo` or something similar that clarifies that it's just an optional part of the class descriptor, and that should also be mentioned in a doc comment.

Reply:
> I assume we just don't currently need to interpret it within the Swift runtime.

Correct. I'll add a better comment and rename this struct.